### ISSUE TITLE
ResourceNotFound Exception (bottombar_tabs_five)

### ISSUE THREAD

**Issue by RoshR** [roughike/BottomBar#785]
I am using the 2.3.1 version. At first the library was working fine but then after a while this this ResourceNotFound Exception of the bottombar_tabs_five.xml file came into picture and the BottomBar.java class fails to instantiate. I have tried a lot of solutions i could find on the net for this, but noting worked.

I even cloned the BottomBar-master project into my AndroidStudio. Even that project shows the same error. Could someone help me out with this??

![2017-05-27_14h12_54](https://cloud.githubusercontent.com/assets/7272580/26519928/9df3e8c8-42e6-11e7-9af5-cfbecbafea32.png)

**Comment by yombunker** [comment]
@RoshR That error just means it can't display itself in the editor, unless you set the tabs trough xml, in which case it will do it. But, is not something it should fail while running.

**Comment by RoshR** [comment]
@yombunker yes, this doesn't fail while running but i cant see the preview. Also i forgot to add this image, the exception which it gives. Edited in the main post.
